backend_windows.go

```diff
@@ -1,6 +1,10 @@
 //go:build windows
 // +build windows
 
+// Windows backend based on ReadDirectoryChangesW()
```

Comment:
RE: Godoc, might be worth changing all instances of "`/some/path/...`" to be "`/some/path/...` or `X:\some\path\...` depending upon platform" or similar

Reply:
I added a note in the "Windows notes" section for this. I think using "/some/path/... or X:\some\path\... depending upon platform" on every case where a path is mentioned would get a bit long, and I don't really expect people to be confused by it. Besides, forward slashes work well on Windows, and have since at least Windows 2000 AFAIK so it's not even needed to use \ instead of /.